### PR TITLE
unemployability dropdown options

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Correction of defect options: When uploading a doc in the "Supporting documents" section, user does not get dropdown options.

NOTE: tempSchema from config/schema is being used and will be changed to fullSchema from vets-json-schema when that PR has merged unemployabilityAttachments in master.

## Screenshots
screen shot 2019-01-10 at 10 04 49 am

## Acceptance criteria
- [ ] Dropdown options appear for user

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by imeecuison** [comment]
NOTE: This PR is on hold until https://github.com/department-of-veterans-affairs/vets-website/pull/9361#pullrequestreview-192220286
merges into master

**Comment by imeecuison** [comment]
screen shot 2019-01-16 at 1 03 21 pm

**Comment by imeecuison** [comment]
@aaronjlech and @zurbergram : This is ready for internally review... again. :)

**Review by zurbergram — CHANGES_REQUESTED** [review]
`git checkout origin/master src/applications/disability-benefits/all-claims/config/8940/index.js` then commit and push again to reset the file to master

**Review by imeecuison — COMMENTED** [review]
Check it again. I fixed it. Put everything back and fixed broken tests. Serious headache. The problem was hospitalizationHistory was already in code but using the old schema. When the new schema was introduced and needed for this particular branch, that page broke everything immediately. Anyway, all tests are passing and hospitalization is back.
@zurbergram

**Review by zurbergram — APPROVED** [review]

**Comment by zurbergram** [comment]
LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

**Comment by imeecuison** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/disability-benefits/all-claims/constants.js [hunk 1]
```diff
@@ -223,6 +223,8 @@ export const PTSD_CHANGE_LABELS = {
 export const ATTACHMENT_KEYS = [
   'form781Upload',
   'form781aUpload',
+  'form8940Upload',
+  'uploaded4192',
```
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/constants.js]: This key may not be consistent with the others e.g. `form4192Upload`

**Review by rianfowler — APPROVED** [review]
LGTM- on minor nit.

File: src/applications/disability-benefits/all-claims/constants.js (hunk 1 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/constants.js]: @zurbergram ^^

Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/constants.js]: changing

**Pull request merged**